packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,67 @@
+# Amount <!-- omit in toc -->
+
+Amount component is a UI element that displays and formats various currency values in a specified format.
+
+<img  src="./amount-thumbnail.png"  width="380" />
+
+## Design
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
+
+| Prop           | Type                                                               | Default     | Description                                                                    | Required |
+| -------------- | ------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------------ | -------- |
+| children       | `string`                                                           | `undefined` | The value to be rendered within the component                                  | ✅       |
+| intent         | `default`, `positive`, `negative`, `notice`, `information`, `blue` | `default`   | The variant of the amount to be rendered                                       |          |
+| contrast       | `low`, `high`                                                      | `low`       | The contrast of the amount to be rendered                                      |          |
+| fontWeight     | `bold`, `regular`                                                  | `bold`      | The weight of the amount to be rendered                                        |          |
+| hasDecimals    | `true`, `false`                                                    | `true`      | Indicates whether the amount has or can have decimals                          |          |
+| hasTextPostfix | `true`, `false`                                                    | `true`      | Indicates whether a text postfix should be used                                |          |
+| size           | `3xl`, `2xl`, `xl`, `large`, `medium`, `small`                     | `3xl`       | The size of the amount to be rendered                                          |          |
+| isStyled       | `true`, `false`                                                    | `true`      | Highlight the main amount by making the prefix symbol and decimal digits small |          |
+| screenSize     | `desktop`, `mobile`                                                | `desktop`   | Provides the size of the component based on the platform                       |          |
```

Comment:
This is for design only. On code we automatically handle this based on our platform specific tokens. So if you choose `font[100]` it will reflect to a different size on web & mobile. 
```suggestion
```